src/effects/effectmanifest.h

```diff
@@ -122,6 +131,8 @@ class EffectManifest {
     bool m_isMasterEQ;
     // This helps us at DlgPrefEQ's basic selection of Filter knob effects
     bool m_isForFilterKnob;
+    // This helps the skin draw knob rings correctly.
+    double m_dSuperKnobScaleStart;
```

Comment:
Please remove m_dSuperKnobScaleStart from the manifest. 
It is a property of each effect parameter, but that is can be postponed to an later PR.